content/docs/sidebar.json

```diff
@@ -87,6 +87,19 @@
         "slug": "what-is-dvc",
         "source": "what-is-dvc.md"
       },
+      {
+        "label": "Basic Concepts",
+        "slug": "glossary",
```

Comment:
BTW. How did we end up with a mix of `glossary` in the URLs but "Basic Concepts" in the nav and title? 😋 

Let's use `concepts` in the slug and URL paths too. We just know this is where we have our glossary (tooltips) internally 🙂 

> Sry, will imply lots of link changes again, prob.

Reply:
Haha, we went back and forth on it. Fixing this first to minimize link changes on the rest.